card/mod/carrierwave/set/abstract/attachment.rb

```diff
@@ -20,7 +20,7 @@ def self.included host_class
   @current_action.update! comment: original_filename
 end
 
-event :validate_file_exist, :validate, on: :save do
+event :validate_file_exist, :validate, on: :create do
```

Comment:
this is another unrelated problem: File cards that where created with :empty_ok cause problems when they are renamed. This solution assumes that once a card was created without a file updates without a file are ok.